school_lunch/controllers/controllers.py

```diff
@@ -86,8 +87,10 @@ def school_kid_add_id(self, kid_id=None, **kw):
             return request.redirect("/school/kids")
         kid_id = int(kid_id)
         d = request.session.get("mykids", [])
-        if kid_id not in d:
+        if int(kid_id) not in d:
```

Comment:
I believe type cast is not needed here, as it's done iin line 88

Reply:
Alright, I will remove the type cast here, it's indeed working without it.
Thanks for reviewing!